packages/@aws-cdk/aws-eks/test/test.user-data.ts

```diff
@@ -155,12 +157,18 @@ function newFixtures(spot = false) {
   const app = new App();
   const stack = new Stack(app, 'my-stack', { env: { region: 'us-west-33' } });
   const vpc = new ec2.Vpc(stack, 'vpc');
+  const cluster = Cluster.fromClusterAttributes(stack, 'cluster', {
+    clusterName: 'my-cluster-name',
+    clusterCertificateAuthorityData: 'CADATA',
+    clusterEndpoint: 'https://mycluster:443',
```

Comment:
Lets add a test with an imported cluster that doesn't specify these properties, to make sure the exceptions are handled properly.